man/geninfo.1

```diff
@@ -123,6 +123,25 @@ Marks the end of a section which is excluded from branch coverage. The current
 line not part of this section.
 .RE
 .br
+LCOV_EXCL_EXCEPTION_BR_LINE
+.RS
+Lines containing this marker will be excluded from exception branch coverage:
+Exception branches will be ignored, but non-exception branches will be 
+included in branch coverage.
```

Comment:
Maybe change the phrasing from "non-exception branches will be included" to "non-exception branches will not be affected" to account for any other reason why the branch data might not be included.

Reply:
Done